src/lab4/vault/Thief.java

```diff
@@ -0,0 +1,39 @@
+package lab4.vault;
+
+import java.util.ArrayList;
+
+public class Thief {
+
+    private String name;
+    private String rank;
+    private int money;
+    private ArrayList<String> passwords = new ArrayList<>();
```

Comment:
e mai bine sa ai in stanga List<String> ca sa schimbi doar in dreapta instanta daca vrei sa folosesti LinkedList in loc de arraylist